kolibri/plugins/document_pdf_render/assets/src/vue/index.vue

```diff
@@ -10,14 +13,61 @@
 <script>
 
   const PDFobject = require('pdfobject');
+
   module.exports = {
+
     props: ['defaultFile'],
+
+    data: {
```

Comment:
This should be a function that returns the object:
```
data: () => ({
    supportsPDFs: PDFobject.supportsPDFs,
}),
```

(This combines the suggestion from below about the direct assignment).